dependent/negative-results.csl

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="en-US">
+ 
+  <info>
+    <title>Negative Results</title>
+    <title-short>Neg Res</title-short>
+    <id>http://www.zotero.org/styles/negative-results</id>
+    <link href="http://www.zotero.org/styles/negative-results" rel="self"/>
+    <link href="http://www.zotero.org/styles/cell" rel="independent-parent"/>
```

Comment:
please add
`<link href="http://www.negative-results.org/index.php/instructions-for-authors/#References" rel="documentation"/>` here